.changeset/big-spiders-develop.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
![image](https://github.com/user-attachments/assets/2b516668-f8e8-4b90-bc48-e7d659feb6dc)
This could be documented, if RAC judge this necessary to document, I feel it'll be a common pattern, accessibility wise it's smart to document this

Reply:
We have never had any examples of interactive elements in a disclosure. For now, we don't support this.
If you know of one, please, let me know.